ticdc: Update cdc server config (#15012)
pingcap/docs: ticdc/ticdc-server-config.md

> The following are configurations in the configuration file of `cdc server`:

Reviewer: ```suggestion
The following describes the configuration file specified by the `config` option in the `cdc server` command:
```

> ```yaml

Author: ```suggestion
```toml
```

  cert-path = ""
  key-path = ""

# The session duration between TiCDC and etcd services, measured in seconds. This parameter is optional and its default value is 10.
capture-session-ttl = 10 # 10s

# The interval at which the Owner module in the TiCDC cluster attempts to push the replication progress. This parameter is optional and its default value is `50000000` nanoseconds (that is, 50 milliseconds). You can configure this parameter in two ways: specifying only the number (for example, configuring it as `40000000` represents 40000000 nanoseconds, which is 40 milliseconds), or specifying both the number and unit (for example, directly configuring it as `40ms`).
owner-flush-interval = 50000000 # 50 ms

# The interval at which the Processor module in the TiCDC cluster attempts to push the replication progress. This parameter is optional and its default value is `50000000` nanoseconds (that is, 50 milliseconds). The configuration method of this parameter is the same as that of `owner-flush-interval`.
processor-flush-interval = 50000000 # 50 ms

> #[log]
> # The output location for internal error logs of the zap log module. This parameter is optional and its default value is "stderr".
> # error-output = "stderr"
> # [log.file]
> #   The maximum size of a single log file, measured in MiB. This parameter is optional and its default value is 300.
> #   max-size = 300 # 300 MiB
> #   The maximum number of days to retain log files. This parameter is optional and its default value is `0`, indicating never to delete.
> #   max-days = 0
> #   The number of log files to retain. This parameter is optional and its default value is `0`, indicating to keep all log files.
> #   max-backups = 0

Author: ```suggestion
# [log]
# # The output location for internal error logs of the zap log module. This parameter is optional and its default value is "stderr".
#   error-output = "stderr"
#   [log.file]
#     # The maximum size of a single log file, measured in MiB. This parameter is optional and its default value is 300.
#     max-size = 300 # 300 MiB
#     # The maximum number of days to retain log files. This parameter is optional and its default value is `0`, indicating never to delete.
#     max-days = 0
#     # The number of log files to retain. This parameter is optional and its default value is `0`, indicating to keep all log files.
#     max-backups = 0
```

#[sorter]
# The size of the shared pebble block cache in the Sorter module for the 8 pebble DBs started by default, measured in MiB. The default value is 128.
# cache-size-in-mb = 128

> # The directory where sorter files are stored relative to the data directory. This parameter is optional and its default value is "/tmp/sorter".

Reviewer: ```suggestion
# The directory where sorter files are stored relative to the data directory (`data-dir`). This parameter is optional and its default value is "/tmp/sorter".
```

# sorter-dir = "/tmp/sorter"

# [kv-client]
#   The number of threads that can be used in a single Region worker. This parameter is optional and its default value is 8.
#   worker-concurrent = 8
#   The number of threads in the shared thread pool of TiCDC, mainly used for processing KV events. This parameter is optional and its default value is 0, indicating that the default pool size is twice the number of CPU cores.
#   worker-pool-size = 0

> #   The retry time of Region connection. This parameter is optional and its default value is `60000000000` nanoseconds (that is, 1 minute). You can configure this parameter in two ways: specifying only the number (for example, configuring it as `50000000` represents 50000000 nanoseconds, which is 50 milliseconds), or specifying both the number and unit (for example, directly configuring it as `50ms`).

Author: ```suggestion
#   The retry duration of Region connection. This parameter is optional and its default value is `60000000000` nanoseconds (that is, 1 minute). You can configure this parameter in two ways: specifying only the number (for example, configuring it as `50000000` represents 50000000 nanoseconds, which is 50 milliseconds), or specifying both the number and unit (for example, directly configuring it as `50ms`).
```
